src/api/spec/components/workflow_run_row_component_spec.rb

```diff
@@ -135,6 +135,7 @@
     let(:request_payload) do
       <<~END_OF_REQUEST
         {
+          "action": "unsupported",
```

Comment:
Push event payloads don't have `action`, so the newly added spec on line 153 should pass without this line.

```suggestion
```

Reply:
That's true, removing